src/api/controllers/notificationController.js

```diff
@@ -0,0 +1,166 @@
+import _ from 'lodash';
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+import sendError from '../../helpers/error.sender';
+import errorMessages from '../../helpers/constants/error.messages';
+import emailUnsubscriptionConfig from '../../helpers/constants/email.unsubscription.config.json';
+
+const { NotificationConfig, Notification } = models;
+
+/**
+ * A function to send the result back for notifications GET requests
+ *
+ * @param {Object} Notifications - The sequelize return value
+ * @param {Object} ResultObject - The result object
+ * @param {Object} Res -  The express result object
+ * @returns {Object} result - The result object
+ */
+const sendNotificationResult = (Notifications, ResultObject, Res) => {
+  if (!_.isEmpty(Notifications)) {
+    // Sending the result
+    ResultObject.status = status.OK;
+    ResultObject.data = Notifications;
+    return Res.status(status.OK).json(ResultObject);
+  }
+  return sendError(status.NOT_FOUND, Res, 'notifications', errorMessages.notificationNotFound);
+};
+
+/**
+ * A class to handle all notifications
+ */
+export default class NotificationController {
+  /**
+   * A method to get user configs
+   * @param  {object} req
+   * @param  {object} res
+   * @return {object} return an object containing set configuration
+   */
+  static async getConfig(req, res) {
+    // Initializing variables
+    const result = {};
+    const {
+      user: { id }
+    } = req.user;
+
+    const userConfig = await NotificationConfig.findOne({
+      userId: id
+    });
+
+    userConfig.config = JSON.parse(userConfig.config);
+
+    // Sending the result
+    result.status = status.OK;
+    result.config = userConfig;
+    return res.status(status.OK).json(result);
```

Comment:
@nkpremices Isn't the team convention to return the status in the body of the result?

Reply:
Yes it is @MemunaHaruna 
I've done it.
Check out the line 52